File: src/main/java/Multiple.java
public class Multiple {

	private static final int[] MULTIPLES = new int[]{ 3, 5 };

	public static void main(String[] args) {

		int sum = 0;
		for (int i = 1; i < 1000; i++){
			for (int multiple: MULTIPLES) {
				if (i % multiple == 0){
					sum += i;
					break;
				}
			}
		}
		System.out.println("sum = " + sum);
	}
}

File: src/main/kotlin/Multiple.kt

private val MULTIPLES = intArrayOf( 3, 5 )

fun main(args: Array<String>) {

    var sum = 0
    for (num in 1 until 1000){
       if ( num.isMultiple(*MULTIPLES) ){
           sum += num
       }
    }
    println("sum = $sum")
}

fun Int.isMultiple(vararg multiples: Int): Boolean{
    for (multiple in multiples){
        if (this % multiple == 0) return true
    }
    return false
}